### PR TITLE
Added Links to The Career support Pages Navbar

### LINKED ISSUE #944: BUG: The Nav Links are Not Working in Career Support page
Hey @thestarsahil 

The nav links of the navbar are not working on the career support page

https://github.com/Counselllor/Counsellor-Web/assets/163530398/285c2928-d3ef-454c-9290-32ac22b335f2

Could you please assign this issue to me under GSSOC24?

### PR DESCRIPTION
**Pull request by RadhikaMalpani1702**
# Description
Hey @thestarsahil 

I have added the links to the Career support page navbar 
Additionally, there was log in button instead of log out so I have removed it and added log out functionality

## Screenshots

https://github.com/Counselllor/Counsellor-Web/assets/163530398/a454dff1-2be2-429f-9650-5383a54c030b

## Checklist

- [X] Tests have been added or updated to cover the changes
- [X] Documentation has been updated to reflect the changes
- [X] Code follows the established coding style guidelines
- [X] All tests are passing

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -106,10 +106,10 @@ const LogoSection = () => (
 const MenuSection = ({ user, handleSignOut, menuOpen }) => (
   <div className={`menu ${menuOpen ? 'show' : ''}`}>
     <ul>
-      <MenuItem href="#">Top Universities</MenuItem>
+      <MenuItem href="/dashboard">Top Universities</MenuItem>
```
Comment (thestarsahil) [inline src/components/Navbar/Navbar.jsx:109]: <MenuItem href="#">Top Universities</MenuItem>

**Review by thestarsahil — CHANGES_REQUESTED** [review]

**Comment by RadhikaMalpani1702** [comment]
@thestarsahil Made the changes

**Comment by RadhikaMalpani1702** [comment]
@thestarsahil Please merge this PR

**Comment by thestarsahil** [comment]
Its Done Already !

**Pull request closed without merging**